DWAPlannerROS:
#DWBLocalPlanner:
# Robot Configuration Parameters
  max_vel_x: 0.5
  min_vel_x: -0.2

  max_vel_y: 0.0
  min_vel_y: 0.0

# The velocity when robot is moving in a straight line
  max_vel_trans:  0.5
  min_vel_trans:  0.00
  max_vel_theta: 0.314
  min_vel_theta: 0.0

  acc_lim_x: 1.0
  acc_lim_y: 0.0
  acc_lim_theta: 3.0
  acc_lim_trans: 1.0
  escape_vel: -0.2
# Goal Tolerance Parametes
  xy_goal_tolerance: 0.3
  yaw_goal_tolerance: 0.08
  latch_xy_goal_tolerance: true

# Forward Simulation Parameters
  sim_time: 5.0
  vx_samples: 10
  vy_samples: 0
  vth_samples: 80
  controller_frequency: 5
# Trajectory Scoring Parameters
  path_distance_bias: 32.0
  goal_distance_bias: 24.0
# occdist_scale:  0.005
  occdist_scale:  0.05
  forward_point_distance: 0.325
  stop_time_buffer: 2.0
  scaling_speed: 0.25
  max_scaling_factor: 0.2
  sim_granularity: 0.2
  angular_sim_granularity: 0.256
  trans_stopped_vel: 0.05
  theta_stopped_vel: 0.05
# Oscillation Prevention Parameters
  oscillation_reset_dist: 0.2
  oscillation_reset_angle: 0.2
# Debugging
  publish_traj_pc : true
  publish_cost_grid_pc: true
  purne_plan: true
